vortexasdk/api/search_result.py

```diff
@@ -15,22 +15,24 @@ def to_list(self) -> List:
         """Represent *records* as a list."""
         return self.records
 
-    def to_df(self, columns: Union[Literal["all"], List[str]]) -> pd.DataFrame:
+    def to_df(
+        self, columns: Literal["all"] | List[str] | None
```

Comment:
I assume `None` and `Literal["all"]` are going to behave in the same way? could we remove `Literal["all"]` maybe? or should we document this in the docstring?

Reply:
Good catch, I actually needed to explicitly handle ensuring that the behaviour was the same for `all` and `None`. I've made this change.